ops/__init__.py

```diff
@@ -12,31 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""The ops library: a Python framework for writing Juju charms.
+"""The API to respond to Juju events and manage the application.
 
-The ops library is a Python framework (`available on PyPI`_) for developing
-and testing Juju charms in a consistent way, using standard Python constructs
-to allow for clean, maintainable, and reusable code.
+This module provides:
 
-A charm is an operator -- business logic encapsulated in a reusable software
-package that automates every aspect of an application's life.
+- :class:`~ops.CharmBase` and :class:`~ops.Object` to register the charm or
+  a charm lib, respectively, with the framework.
+- :class:`~ops.framework.EventBase` class and individual event types, like
+  :class:`~ops.ActionEvent` class.
+- :class:`~ops.Framework` class, accessible as ``self.framework`` in a charm,
+  the main interface for the charm to `ops` library infrastructure, including:
 
-Charms written with ops support Kubernetes using Juju's "sidecar charm"
-pattern, as well as charms that deploy to Linux-based machines and containers.
+  - :meth:`~ops.Framework.on` shorthand property used to
```

Comment:
I think we should go over this section after our daily sync. I think there's a bit too much here and it could be organised a bit better. Also, some things seem incorrect -- isn't `on` on the CharmBase, rather than the Framework?

Reply:
done in live review.